config/sidekiq_scheduler.yml

```diff
@@ -0,0 +1,5 @@
+transmit_education_spool_files:
+  cron: "0 3 * * * America/New_York"
+  class: EducationForm::CreateDailySpoolFiles
+  queue: default
```

Comment:
No, it's not strictly necessary. I thought it was required for it to show the queue name in the Web UI, but I guess not.